pkg/utils/containerValidation.go

```diff
@@ -0,0 +1,86 @@
+package utils
+
+import (
+	"reflect"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
+)
+
+// In this function we are comparing the containerIDs before chaos and after chaos
+// ContainerID before chaos has to be recorded in the BDD only and passed here as an argument
+// In this function we are getting containerID after chaos and checking if it changed or not.
+// If the containerID will change then it will return true else false value.
+func CompareContainerId(containerIdBefore [9]string, podNS string, podLabel string, client *kubernetes.Clientset) (bool, error) {
```

Comment:
Can you please put podLabel and podNS in a structure i.e podDetails. It will reduce the no. of args?